### ISSUE TITLE
Ability to import data from PDF

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#3605]
We support [populating PDFs with form data](https://doc.orbeon.com/form-builder/pdf-production.html), but the reverse, that is creating or updating form data based on data in PDFs.

- Importing data from PDF templates might not be too hard since those already have AcroFields, and we have a mapping between the AcroFields and the fields of forms created with Form Builder.
- For the automatic PDF, we would first need to generate AcroFields.

Also:

- To support updating data (vs. creating new data), we'll want to store the `document_id` somewhere in the PDF.
- We should at the minimum have an API for this, and if possible also a UI.

[+1 from user](https://mail.google.com/mail/u/0/#inbox/FMfcgxmZVXtjfFkJntWQqQPvgJhHKfMS)

**Comment by ebruchez** [comment]
Good points. Also, data should be imported as if typed by the user, to ensure all security constraints are met.

**Comment by avernet** [comment]
Good point, I added that in the description, as well as a note to say that support for adding / removing iterations is out of scope. (And I'm not even sure that companies other than Adobe can generate PDFs that supports those features.)

**Comment by avernet** [comment]
[+1 from user](http://discuss.orbeon.com/Import-fillable-pdf-into-orbeon-form-td4662642.html)
